benchmarking/src/evaluator.py

```diff
@@ -112,6 +132,14 @@ def main():
             help="The absolute path to the dataset to be used for running the custom performance evaluation."
         )
 
+        parser.add_argument(
+            '--save-llm-responses',
+            type=str2bool,
```

Comment:
Didn't realize you could put a function as a type here. Pretty cool! Have you tested to make sure it functions as intended?

Reply:
hi @snova-connorm, yes. Usually for CLI boolean arg parameters, you include or not a flag in the command e.g. `sh my_process.sh -flag`. Since in this we want to use a true/false arg parameter, then I figure out that way, and sure it's working :)